cctools/libstuff/guess_short_name.c

```diff
@@ -59,6 +59,14 @@ static char *look_back_for_slash(
  * If the name of the dynamic library is none of the forms above then NULL is
  * returned.
  */
+/*
+ * MDT 20190119 rdar://12400897
```

Comment:
Has this one been posted to [OpenRadar](https://openradar.appspot.com/page/1)? I looked for it there, but couldn't find it...